add current URL as default (#968)
inrupt/solid-client-authn-js: CHANGELOG.md

## Unreleased

> - Just as a convenience for the developer, on the `handleIncomingRedirect`
>   method of the `Session` class in the browser package, set the current browser
>   URL as the default value.

Reviewer: ```suggestion

### New features

#### browser

- `handleIncomingRedirect` uses the current browser URL as a default value.
```
